README.md

```diff
@@ -1 +1 @@
-# JS-coding-exercise-1
+# JS-coding-exercise-3
```

Comment:
you wouldn't typically change the README.md like this.  When you merge this to master, it will only contain JS-coding-exercise-3.  It would be better to add the names as a new line each time you start them.   You could create a roadmap where you list all of the exercises and then put a checkmark next to them as you complete them.  That would communicate better to people who might view the readme